cognite/v05/depthseries.py

```diff
@@ -16,12 +16,41 @@
 import cognite._utils as _utils
 import cognite.config as config
 
-from cognite.v05 import timeseries
+import cognite.v05.timeseries as ts
 from cognite.v05.dto import LatestDatapointResponse, \
     DatapointDepth, \
-    TimeSeries, TimeSeriesResponse, Datapoint
+    TimeSeries, TimeSeriesResponse, Datapoint, TimeseriesWithDatapoints
 
 
+def post_multitag_datapoints(depthseries_with_datapoints: List[TimeseriesWithDatapoints],**kwargs):
+    '''Insert data into multiple depthseries.
+
+        Args:
+            depthseries_with_datapoints (List[v05.dto.DepthseriesWithDatapoints]): The depthseries with data to insert.
+
+        Keyword Args:
+            api_key (str): Your api-key.
+
+            project (str): Project name.
+
+        Returns:
+            An empty response.
+        '''
+    msIncrement: int = 1000
```

Comment:
Could this be a constant somewhere else? Since I see we have it at least two places.

Reply:
Good point